src/utilities/s3Storage.ts

```diff
@@ -0,0 +1,48 @@
+import S3, { ClientConfiguration } from "aws-sdk/clients/s3";
```

Comment:
src/utilities/s3Storage.ts

I wonder if we need a new directory for modules perhaps src/storage/s3.ts or something?